act/tests/test_discovery.py

```diff
@@ -55,17 +54,18 @@ def test_get_armfile():
     if not os.path.isdir(os.getcwd() + '/data/'):
         os.makedirs(os.getcwd() + '/data/')
 
-    uname = os.getenv('ARM_USERNAME')
-    token = os.getenv('ARM_PASSWORD')
+    # Place your username and token here
+    username = os.environ['ARM_USERNAME']
```

Comment:
Can you change this back to `os.getenv('variable')`? For example,

```python
var = os.getenv('SAMPLE_SECRET')
```

I tried this out in this repo https://github.com/mgrover1/test-actions, it seemed to work (although it hid the secret from printing  - which is to be expected)

Sample output
```bash
Run python print_env_var.py
[7](https://github.com/mgrover1/test-actions/runs/5649861907?check_suite_focus=true#step:4:7)
***
```